src/plugins/data/public/search/aggs/buckets/create_filter/range.test.ts

```diff
@@ -75,25 +85,28 @@ describe('AggConfig Filters', () => {
         ],
         {
           typesRegistry: mockAggTypesRegistry([getRangeBucketAgg(aggTypesDependencies)]),
-          fieldFormats: aggTypesDependencies.getInternalStartServices().fieldFormats,
         }
       );
     };
 
     test('should return a range filter for range agg', () => {
       const aggConfigs = getAggConfigs();
-      const filter = createFilterRange(aggConfigs.aggs[0] as IBucketAggConfig, {
-        gte: 1024,
-        lt: 2048.0,
-      });
+      const filter = createFilterRange(getInternalStartServices)(
+        aggConfigs.aggs[0] as IBucketAggConfig,
+        {
+          gte: 1024,
+          lt: 2048.0,
+        }
+      );
 
+      expect(fieldFormats.deserialize).toHaveBeenCalledTimes(1);
       expect(filter).toHaveProperty('range');
       expect(filter).toHaveProperty('meta');
       expect(filter.meta).toHaveProperty('index', '1234');
       expect(filter.range).toHaveProperty('bytes');
       expect(filter.range.bytes).toHaveProperty('gte', 1024.0);
       expect(filter.range.bytes).toHaveProperty('lt', 2048.0);
-      expect(filter.meta).toHaveProperty('formattedValue', '≥ 1,024 and < 2,048');
+      expect(filter.meta).toHaveProperty('formattedValue');
```

Comment:
We are no longer testing for specific values here since the mocks aren't returning any real values, but I don't think they were the right thing to test here anyway. Instead I'm just making sure a formattedValue is included, and also verifying that `fieldFormats.deserialize` was called as expected. Whether the values are deserialized correctly is something to test in field formats.